e2e/deployers/applicationset.go

```diff
@@ -10,24 +10,25 @@ import (
 
 type ApplicationSet struct{}
 
+// Deploy creates an ApplicationSet on the hub cluster, creating the workload on one of the managed clusters.
 func (a ApplicationSet) Deploy(ctx types.Context) error {
 	name := ctx.Name()
 	log := ctx.Logger()
-	namespace := util.ArgocdNamespace
+	hubNamespace := ctx.ManagementNamespace()
```

Comment:
I like the names ManagementNamespace and ApplicationNamespace. I don't understand why we are calling it hubNamespace here or clusterNamespace for applicationNamespace below. Whether is the namespace is on the hub or cluster is dependent on the operation and it should be clear based on the operation. I would prefer the variable also to be called managementNamespace.

Reply:
Adding hub/cluster makes it more clear for someone which new to ramen. I can change it.

Reply:
Replaced with management/app.